### PR TITLE
Update moengage-event-streaming.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by kevinpagtakhan** [comment]
@justin-fiedler @bgiori can we get this reviewed? thank you!

**Comment by Brandoncharleskhoo1** [comment]
Hi @justin-fiedler can I please get a review when you get a chance? no rush.

File: docs/data/destinations/moengage-event-streaming.md [hunk 1]
```diff
@@ -46,13 +46,18 @@ Create an access token in Extole. You need this to complete the Amplitude setup.
...
-8. Use the Event Properties filter to select which Event Properties you would like to send.
-9. When finished, save your work.
+7. Toggle the **Send events** filter to select the events to send. You can send all events, but Amplitude recommends choosing the most important ones.
+8. Use the **Event Properties** filter to select which Event Properties you would like to send.
+9. Under **Send Users**, make sure the toggle is enabled ("Users are sent to MoEngage") if you want to stream users and their properties to MoEngage. When enabled, users are automatically created or updated in MoEngage when an event is sent to Amplitude.
+10. When finished, save your work.
 
+## View forwarded events from Amplitude in MoEngage
+1. Log into MoEngage.
+2. Click on **Events** tab.
```
Comment (markzegarelli) [inline docs/data/destinations/moengage-event-streaming.md:56]:
```suggestion
2. Navigate to the the **Events** tab.
```

File: docs/data/destinations/moengage-event-streaming.md [hunk 2]
```diff
@@ -46,13 +46,18 @@ Create an access token in Extole. You need this to complete the Amplitude setup.
...
+3. Click on **Custom Events** to view the log of events.
```
Comment (markzegarelli) [inline docs/data/destinations/moengage-event-streaming.md:57]:
```suggestion
3. Click **Custom Events** to view the log of events.
```

File: docs/data/destinations/moengage-event-streaming.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ Keep these things in mind when sending events to [MoEngage](https://www.moengage
 - You need a MoEngage account to enable this integration.
 - Relevant limits for MoEngage events are:
     - 10k events per app per minute
-- Amplitude sends selected user, event, and group properties along with the event.
-
+- Amplitude sends selected user and event properties along with the event.
+- This integration supports user identify forwarding. This means when **Send Users** is enabled during the setup, user updates will be sent to MoEngage every time any user property is changed in Amplitude.
```
Comment (markzegarelli) [inline docs/data/destinations/moengage-event-streaming.md:22]:
```suggestion
- This integration supports user identify forwarding. This means when you enable **Send Users**  during setup, user updates are sent to MoEngage every time any user property changes in Amplitude.
```

Present tense, active voice.

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/destinations/moengage-event-streaming.md (hunk 1 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/moengage-event-streaming.md:56]: Ooops I should probarbly change my template to make sure I incorporate these changes. Thank you!

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**